File: benchmarks/javascript/fibonacci.js
function fib(n) {
    if (n <= 1) {
        return n;
    }
    return fib(n - 1) + fib(n - 2);
}

const N = 30;
const startTime = Date.now();
const result = fib(N);
const endTime = Date.now();

console.log(`Fibonacci(${N}) = ${result}`);
console.log(`Time: ${endTime - startTime}ms`);

File: benchmarks/python/fibonacci.py
import time

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

N = 30
start_time = time.time()
result = fib(N)
end_time = time.time()

print(f"Fibonacci({N}) = {result}")
print(f"Time: {(end_time - start_time) * 1000:.2f}ms")